tools/tgs_scripts/InstallDeps.sh

```diff
@@ -28,6 +28,15 @@ fi
 # install cargo if needed
 if ! [ -x "$has_cargo" ]; then
 	echo "Installing rust..."
+	if ! [ -x "$has_sudo" ]; then
+		dpkg --add-architecture i386
+		apt-get update
+		apt-get install -y g++-multilib libclang-dev:i386
+	else
+		sudo dpkg --add-architecture i386
+		sudo apt-get update
+		sudo apt-get install -y g++-multilib libclang-dev:i386
```

Comment:
Why do you think it needs libclang-dev:i386?

it doesn't appear to be installed on campbell.

I suspect I went wrong with the deps and it only needs g++-mulitlib

Reply:
e.g only the 64bit version of clang is installed.

Reply:
Yeah i just tested this in the docker env, you only need g++-multilib and clang itself, not the 32bit version of clang.

root@b18083c8f571:/dreamluau# file target/i686-unknown-linux-gnu/release/libdreamluau.so
target/i686-unknown-linux-gnu/release/libdreamluau.so: ELF 32-bit LSB shared object, Intel 80386, version 1 (SYSV), dynamically linked, BuildID[sha1]=7bcbe27585af61e4ce5fa62ff80505a3cb9c296a, not stripped

Reply:
> Why do you think it needs libclang-dev:i386?
> 
> it doesn't appear to be installed on campbell.
> 
> I suspect I went wrong with the deps and it only needs g++-mulitlib

because i got a travis ci error the commit before and google suggested that???

like why else?

Reply:
ratio + l + googled

Reply:
anyway you dont need it, just clang 64bit.

Reply:
Yeah give it a shot please